book/10-git-internals/sections/objects.asc

```diff
@@ -139,6 +139,12 @@ $ git cat-file -p master^{tree}
 The `master^{tree}` syntax specifies the tree object that is pointed to by the last commit on your `master` branch.
 Notice that the `lib` subdirectory isn't a blob but a pointer to another tree:
 
+**NOTE**: while using `master^{tree}` syntax if you have error try the following
```

Comment:
Instead of just setting this in bold-face, let's use the Asciidoc "note" feature:

```asciidoc
[NOTE]
====
(Note content goes here)
====
```

This will render much nicer.

In addition, let's change the prose here to read like this:

```asciidoc
Depending on what shell you use, you may encounter errors when using the `master^{tree}` syntax.
In PowerShell or CMD on Windows, the `^` character is used for escaping, so you have to double it to avoid this: `git cat-file -p master^^{tree}`.
If you're using ZSH, the `^` character is used for globbing, so you have to enclose the whole expression in quotes: `git cat-file -p "master^{tree}"`.
```

Or something like that. It's a bit more idiomatically English.